modules/gdnative/gdnative_api.json

```diff
@@ -6580,7 +6580,23 @@
         "major": 1,
         "minor": 1
       },
-      "next": null,
+      "next": {
```

Comment:
Im trying to make this work too, in this line a sub android api major:1 minor:2 is introduced, and the godot_android_get _display_rotation is defined under this sub api,
in the original work PR https://github.com/godotengine/godot/pull/26221 instead the original android api major:1 minor:1 is modified and the godot_android_get_display_rotation is defined at the same level of the other android api call,
so I think that is an error